### PR TITLE
feat: handle 401 Unauthorized response

### PR DESCRIPTION
**Pull request by tracy-french**
# Description

This change triggers an automatic logout when a 401 Unauthorized HTTP status code is detected. The Amplify `signOut` method is utilized to trigger the process.

## Legal

This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

File: apps/client/src/serviceWorkerRegistration.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export function register() {
+  const swUrl = `${process.env.PUBLIC_URL}/service-worker.js`;
```
Comment (chejimmy) [inline apps/client/src/serviceWorkerRegistration.ts:2]: does it work with the path only: `/service-worker.js`?

File: apps/client/public/service-worker.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+self.addEventListener('install', () => {
```
Comment (chejimmy) [inline apps/client/public/service-worker.js:1]: this cannot be `.ts`?

Comment (tracy-french) [reply apps/client/public/service-worker.js:1]: It can, but we need to transpile it to JS.

**Review by chejimmy — APPROVED** [review]
LGTM! exciting technologies to serve customers

**Pull request merged**